<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mAPreferencesWindow</class>
 <widget class="QDialog" name="mAPreferencesWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>435</width>
    <height>367</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>435</width>
    <height>367</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>435</width>
    <height>367</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>521</width>
    <height>417</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Preferences</string>
  </property>
  <property name="windowIcon">
   <iconset resource="miniAudicle.qrc">
    <normaloff>:/icon/miniAudicle.png</normaloff>:/icon/miniAudicle.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>5</number>
   </property>
   <property name="margin">
    <number>5</number>
   </property>
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="minimumSize">
      <size>
       <width>425</width>
       <height>325</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>425</width>
       <height>325</height>
      </size>
     </property>
     <property name="currentIndex">
      <number>3</number>
     </property>
     <widget class="QWidget" name="audioTab">
      <attribute name="title">
       <string>Audio</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <property name="spacing">
        <number>12</number>
       </property>
       <property name="leftMargin">
        <number>12</number>
       </property>
       <property name="rightMargin">
        <number>12</number>
       </property>
       <property name="bottomMargin">
        <number>12</number>
       </property>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,0,0">
         <item>
          <widget class="QCheckBox" name="enableNetworkVM">
           <property name="text">
            <string>Accept network VM commands</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::MinimumExpanding</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QCheckBox" name="enableAudio">
           <property name="text">
            <string>Enable Audio</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <property name="spacing">
          <number>-1</number>
         </property>
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Audio output:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="audioOutput">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>1</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Audio input:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="audioInput">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>1</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_7">
         <item>
          <widget class="QLabel" name="label_3">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
             <horstretch>1</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>108</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string>Output channels:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="outputChannels">
           <property name="maximumSize">
            <size>
             <width>100</width>
             <height>16777215</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="label_4">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
             <horstretch>1</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>100</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string>Input channels:</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="inputChannels">
           <property name="maximumSize">
            <size>
             <width>100</width>
             <height>16777215</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_8">
         <item>
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>Sample rate:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="sampleRate"/>
         </item>
         <item>
          <spacer name="horizontalSpacer_4">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>Buffer size:</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="bufferSize">
           <property name="currentIndex">
            <number>3</number>
           </property>
           <item>
            <property name="text">
             <string>64</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>128</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>256</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>512</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>1024</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>2048</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>10</width>
           <height>10</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="editingTab">
      <attribute name="title">
       <string>Editing</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="spacing">
        <number>12</number>
       </property>
       <property name="leftMargin">
        <number>12</number>
       </property>
       <property name="rightMargin">
        <number>12</number>
       </property>
       <property name="bottomMargin">
        <number>12</number>
       </property>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_9">
         <item>
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>Editor Font:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QFontComboBox" name="font">
           <property name="currentFont">
            <font>
             <family>Courier New</family>
             <pointsize>12</pointsize>
            </font>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="fontSize">
           <property name="minimum">
            <number>8</number>
           </property>
           <property name="maximum">
            <number>36</number>
           </property>
           <property name="value">
            <number>10</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_8">
           <property name="text">
            <string>pt</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QCheckBox" name="enableSyntaxColoring">
         <property name="text">
          <string>Enable syntax coloring</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_10">
         <item>
          <spacer name="horizontalSpacer_6">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Fixed</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QComboBox" name="syntaxColoringType">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>1</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="syntaxColoringChangeButton">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
             <horstretch>1</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="maximumSize">
            <size>
             <width>125</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="palette">
            <palette>
             <active>
              <colorrole role="Button">
               <brush brushstyle="SolidPattern">
                <color alpha="255">
                 <red>0</red>
                 <green>0</green>
                 <blue>0</blue>
                </color>
               </brush>
              </colorrole>
             </active>
             <inactive>
              <colorrole role="Button">
               <brush brushstyle="SolidPattern">
                <color alpha="255">
                 <red>0</red>
                 <green>0</green>
                 <blue>0</blue>
                </color>
               </brush>
              </colorrole>
             </inactive>
             <disabled>
              <colorrole role="Button">
               <brush brushstyle="SolidPattern">
                <color alpha="255">
                 <red>0</red>
                 <green>0</green>
                 <blue>0</blue>
                </color>
               </brush>
              </colorrole>
             </disabled>
            </palette>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_5">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QCheckBox" name="editorUsesTabs">
         <property name="text">
          <string>Editor uses tabs, not spaces</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_12">
         <item>
          <spacer name="horizontalSpacer_8">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Fixed</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="label_9">
           <property name="text">
            <string>Tab width:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="tabWidth">
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>12</number>
           </property>
           <property name="value">
            <number>4</number>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_7">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>10</width>
           <height>10</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="chuginsTab">
      <attribute name="title">
       <string>ChuGins</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <property name="spacing">
        <number>12</number>
       </property>
       <property name="leftMargin">
        <number>12</number>
       </property>
       <property name="rightMargin">
        <number>12</number>
       </property>
       <property name="bottomMargin">
        <number>12</number>
       </property>
       <item>
        <widget class="QCheckBox" name="enableChugins">
         <property name="text">
          <string>Enable ChuGins</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QListWidget" name="chuginsList"/>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <property name="spacing">
          <number>0</number>
         </property>
         <item>
          <widget class="QToolButton" name="removeChuginButton">
           <property name="font">
            <font>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>-</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="addChuginButton">
           <property name="font">
            <font>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>+</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_9">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="miscTab">
      <attribute name="title">
       <string>Miscellaneous</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <property name="spacing">
        <number>12</number>
       </property>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QLabel" name="label_10">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>3</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Current Directory:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="currentDirectoryButton">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
             <horstretch>1</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Select...</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QLineEdit" name="currentDirectory">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_3">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>-1</number>
     </property>
     <item>
      <widget class="QPushButton" name="restoreDefaultsButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Restore Defaults</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::MinimumExpanding</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="okButton">
       <property name="text">
        <string>OK</string>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="miniAudicle.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>okButton</sender>
   <signal>clicked()</signal>
   <receiver>mAPreferencesWindow</receiver>
   <slot>ok()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>456</x>
     <y>390</y>
    </hint>
    <hint type="destinationlabel">
     <x>260</x>
     <y>204</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>mAPreferencesWindow</receiver>
   <slot>cancel()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>326</x>
     <y>390</y>
    </hint>
    <hint type="destinationlabel">
     <x>260</x>
     <y>204</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>restoreDefaultsButton</sender>
   <signal>clicked()</signal>
   <receiver>mAPreferencesWindow</receiver>
   <slot>restoreDefaults()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>80</x>
     <y>390</y>
    </hint>
    <hint type="destinationlabel">
     <x>260</x>
     <y>204</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>audioInput</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>mAPreferencesWindow</receiver>
   <slot>SelectedAudioInputChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>258</x>
     <y>132</y>
    </hint>
    <hint type="destinationlabel">
     <x>220</x>
     <y>204</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>audioOutput</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>mAPreferencesWindow</receiver>
   <slot>SelectedAudioOutputChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>258</x>
     <y>103</y>
    </hint>
    <hint type="destinationlabel">
     <x>220</x>
     <y>204</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>syntaxColoringChangeButton</sender>
   <signal>clicked()</signal>
   <receiver>mAPreferencesWindow</receiver>
   <slot>syntaxColoringChangeColor()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>149</y>
    </hint>
    <hint type="destinationlabel">
     <x>220</x>
     <y>204</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>syntaxColoringType</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>mAPreferencesWindow</receiver>
   <slot>syntaxColoringTypeChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>118</x>
     <y>148</y>
    </hint>
    <hint type="destinationlabel">
     <x>220</x>
     <y>204</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>addChuginButton</sender>
   <signal>clicked()</signal>
   <receiver>mAPreferencesWindow</receiver>
   <slot>addChugin()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>38</x>
     <y>347</y>
    </hint>
    <hint type="destinationlabel">
     <x>220</x>
     <y>204</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>removeChuginButton</sender>
   <signal>clicked()</signal>
   <receiver>mAPreferencesWindow</receiver>
   <slot>removeChugin()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>66</x>
     <y>347</y>
    </hint>
    <hint type="destinationlabel">
     <x>220</x>
     <y>204</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>currentDirectoryButton</sender>
   <signal>clicked()</signal>
   <receiver>mAPreferencesWindow</receiver>
   <slot>changeCurrentDirectory()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>328</x>
     <y>63</y>
    </hint>
    <hint type="destinationlabel">
     <x>200</x>
     <y>181</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>ok()</slot>
  <slot>cancel()</slot>
  <slot>restoreDefaults()</slot>
  <slot>SelectedAudioOutputChanged()</slot>
  <slot>SelectedAudioInputChanged()</slot>
  <slot>syntaxColoringTypeChanged()</slot>
  <slot>syntaxColoringChangeColor()</slot>
  <slot>addChugin()</slot>
  <slot>removeChugin()</slot>
  <slot>changeCurrentDirectory()</slot>
 </slots>
</ui>
